src/components/toggle/_toggle.scss

```diff
@@ -36,7 +37,8 @@
       height: 4px;
       top: -2px;
       background-color: $ui-05;
-      transition: $transition--base $bx--standard-easing;
+      transition-duration: 60ms; // shix change
+      transition-timing-function: $bx--ease-out; // shix change
```

Comment:
Could you remove the comments from here? Or if this PR is still a WIP, could you add that to the title / label? Thanks Wonil!